lib/databases/timescale/lootboxes/testnet-users.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+package lootboxes
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/fluidity-money/fluidity-app/common/ethereum/fluidity"
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/timescale"
+)
+
+const (
+	// TableTestnetAddress that contains a list of addresses used in the testnet
+	TableTestnetAddress = `testnet_address`
```

Comment:
I think it's wise to preface that this is lootbox specific so we don't have some weird confusion later down the road